apps/civicsignalblog/migrations/20241122_054517_migration.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrateUpArgs, MigrateDownArgs } from "@payloadcms/db-mongodb";
+import mongoose from "mongoose";
+
+const mongoURL = process.env.MONGO_URL;
+const oldGlobalConfigType = "settings-site";
+const newGlobalConfigType = "settings-research-site";
+
+export async function up({ payload }: MigrateUpArgs): Promise<void> {
+  await mongoose.connect(mongoURL);
+  const db = mongoose.connection.db;
+
+  const existingGlobalConfig = await db.collection("globals").findOne({
+    globalType: oldGlobalConfigType,
+  });
+
+  if (!existingGlobalConfig) {
+    payload.logger.error(
+      `No existing global settings [${oldGlobalConfigType}] config found, exiting...`,
+    );
+    return;
+  }
```

Comment:
Should this **really** be an error (`payload.logger.error`)?

Reply:
Hmm, we can keep it as `info` 🤔